official/resnet/resnet.py

```diff
@@ -94,16 +94,23 @@ def process_record_dataset(dataset, is_training, batch_size, shuffle_buffer,
 
 
 ################################################################################
-# Functions building the ResNet model.
+# Convenience functions building the ResNet model.
 ################################################################################
-def batch_norm_relu(inputs, training, data_format):
-  """Performs a batch normalization followed by a ReLU."""
+def batch_norm(inputs, training, data_format):
+  """Performs a batch normalization using a standard set of parameters"""
   # We set fused=True for a significant performance boost. See
   # https://www.tensorflow.org/performance/performance_guide#common_fused_ops
-  inputs = tf.layers.batch_normalization(
+  return tf.layers.batch_normalization(
       inputs=inputs, axis=1 if data_format == 'channels_first' else 3,
       momentum=_BATCH_NORM_DECAY, epsilon=_BATCH_NORM_EPSILON, center=True,
       scale=True, training=training, fused=True)
+
+
+def batch_norm_relu(inputs, training, data_format, intermediate_addition=None):
+  """Performs a batch normalization followed by a ReLU."""
```

Comment:
Docstring with Args and Returns here would be nice-- what is intermediate_addition?